### PR TITLE
Fix the payment amount width for several type of numbers.

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

We used to handle correctly the amount between £1 - £99. This PR adds the more cases.

### Before
picture 344

picture 345

#### £5
picture 347

#### £50
picture 348

#### £500
picture 349

#### £500.20
picture 350

#### £1000

picture 351

#### £1000.20
picture 352

### CODE REVIEW

File: assets/components/paymentAmount/paymentAmount.jsx [hunk 1]
```diff
@@ -21,9 +23,19 @@ type PropTypes = {
 
 export default function PaymentAmount(props: PropTypes) {
 
-  const wideValue = props.amount.toString().length > 2;
-  const className = `component-payment-amount ${wideValue ? wideClass : ''}`;
-  const printedAmount = wideValue ? props.amount.toFixed(2) : props.amount;
+  const addDecimal = props.amount % 1 > 0;
```
Comment (davidfurey) [inline assets/components/paymentAmount/paymentAmount.jsx]: 👍  I've never `% 1` be useful before!

File: assets/pages/monthly-contributions/monthlyContributions.jsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ const store = createStore(reducer, {
 
 // Retrieves the contrib amount from the url and sends it to the redux store.
 const contributionAmount = getQueryParameter('contributionValue', '5');
-
+console.log(contributionAmount);
```
Comment (davidfurey) [inline assets/pages/monthly-contributions/monthlyContributions.jsx]: I think this can go?

Comment (svillafe) [inline assets/pages/monthly-contributions/monthlyContributions.jsx]: 😰😱 Nice catch

File: assets/components/paymentAmount/paymentAmount.jsx (hunk 1 above)
Comment (svillafe) [inline assets/components/paymentAmount/paymentAmount.jsx]: It returns the decimal part :)

**Comment by superfrank** [comment]
500.20/1000.20 don't appear to be centered - needs some space on the right. Then 👍

**Comment by svillafe** [comment]
Fix `paddng-right` for `500.20` and `1000.20` cases.
picture 354
picture 355

**Pull request merged**